mitodl/mitxpro
# Save voucher pdf uploads to S3

#### Pre-Flight checklist


#### What are the relevant tickets?
#536 

#### What's this PR do?
Saves uploaded voucher pdf's (locally or to S3 depending on settings)

#### How should this be manually tested?
- Set the following .env variables based on CI:
  ```python
  AWS_ACCESS_KEY_ID
  AWS_SECRET_ACCESS_KEY
  AWS_STORAGE_BUCKET_NAME
  MITXPRO_USE_S3=True
  ```
- Upload a voucher (I can send you a sample) at `http://xpro.odl.local:8053/voucher/upload/`
- Check that the file exists in S3:
  ```python
  s3 = boto3.resource("s3")
  bucket = s3.Bucket(settings.AWS_STORAGE_BUCKET_NAME)
  [obj for obj in bucket.objects.filter(Prefix='vouchers/')]
  ```
